Marsiple/App/New Group/AlbumsViewController.swift

```diff
@@ -9,12 +9,17 @@
 import SnapKit
 
 class AlbumsViewController: UIViewController {
-    private let albums = [Album](repeating: Album(images: [#imageLiteral(resourceName: "Image1"), #imageLiteral(resourceName: "Image2"), #imageLiteral(resourceName: "Image3"), #imageLiteral(resourceName: "Image4"), #imageLiteral(resourceName: "Image5"), #imageLiteral(resourceName: "Image1"), #imageLiteral(resourceName: "Image2"), #imageLiteral(resourceName: "Image3"), #imageLiteral(resourceName: "Image4")]), count: 10) // TODO: Popoulate with real data
-    private lazy var photoCollectionViewControllers = populatePhotoCollectionViewControllers()
+    private var albums = [Album]()
```

Comment:
Lets add a `didSet` here, so that, when the `albums` are loaded, we set the `photoCollectionViewControllers`. 

A much simpler and less error-prone approach that the one you used with `func tableView(_ tableView: UITableView, willDisplay cell: UITableViewCell, forRowAt indexPath: IndexPath)`.

Reply:
It was done like so at first, but I changed it because there are so many requests and it's pretty slow which causes sessions to timeout so I don't get all the images. This way `PhotoCollectionViewControllers` are initialized as they are shown, so there is less requests, and if you don't scroll to quickly there will be no session timeouts.